mmv1/third_party/terraform/utils/validation.go

```diff
@@ -82,7 +82,7 @@ var rfc1918Networks = []string{
 	"192.168.0.0/16",
 }
 
-func validateGCPName(v interface{}, k string) (ws []string, errors []error) {
+func validateGCEName(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
It would be good to have a comment here to document what the source of this regex is. Something like:

```suggestion
// validateGCEName ensures that a field matches the requirements for Compute Engine resource names
// https://cloud.google.com/compute/docs/naming-resources#resource-name-format
func validateGCEName(v interface{}, k string) (ws []string, errors []error) {
```